@here/harp-examples/src/hello_custom_theme.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GeoCoordinates } from "@here/harp-geoutils";
+import { MapControls, MapControlsUI } from "@here/harp-map-controls";
+import { CopyrightElementHandler, CopyrightInfo, MapView } from "@here/harp-mapview";
+import { APIFormat, OmvDataSource } from "@here/harp-omv-datasource";
+import { accessToken } from "../config";
+
+/**
+ * This example shows how to customize buildin `Berlin` theme using [Theme] `definition` mechanism.
+ *
+ * > NOTE:
+ * > This example focuses on [[Theme]] customization. Please refer to [[HelloWorldExample]] for
+ * > introduction how to use [[MapView]]
+ *
+ * [Theme]s can inherit from another theme by using `extens: URL` property:
```

Comment:
extends
("Code Spell Checker" extension from streetsidesoftware.code-spell-checker is what I am using)